package.json

```diff
@@ -21,20 +24,59 @@
   },
   "homepage": "https://github.com/cachilders/backpat#readme",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-plugin-istanbul": "^3.0.0",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-register": "^6.18.0",
     "chai": "^3.5.0",
     "chai-spies": "^0.7.1",
     "coveralls": "^2.11.15",
+    "cross-env": "^3.1.3",
     "eslint": "^3.8.1",
     "eslint-plugin-flowtype": "^2.25.0",
+    "ghooks": "^1.3.2",
     "mocha": "^3.1.2",
     "npm-run-all": "^3.1.2",
-    "nyc": "^10.0.0"
+    "nyc": "^10.0.0",
+    "rimraf": "^2.5.4"
   },
   "nyc": {
     "check-coverage": true,
     "functions": 90,
     "lines": 90,
-    "statements": 90
+    "statements": 90,
+    "reporter": [
+      "text",
+      "lcov"
+    ],
+    "instrument": false,
+    "sourceMap": false,
+    "require": [
+      "babel-register"
+    ],
+    "include": [
+      "src"
+    ]
+  },
+  "babel": {
+    "presets": [
+      "es2015"
+    ],
+    "env": {
+      "test": {
+        "plugins": [
+          "istanbul"
+        ]
+      }
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "config": {
+    "ghooks": {
+      "pre-commit": "npm run validate"
```

Comment:
Before each commit, run npm run validate. If exits with 2, abort commit. You can escape by writing git commit -m '<message>' --no-verify